packages/main/src/TableGroupRow.js

```diff
@@ -0,0 +1,153 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { fetchI18nBundle, getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import TableGroupRowTemplate from "./generated/templates/TableGroupRowTemplate.lit.js";
+import TableMode from "./types/TableMode.js";
+
+// Texts
+import {
+	TABLE_GROUP_ROW_ARIA_LABEL,
+} from "./generated/i18n/i18n-defaults.js";
+
+// Styles
+import styles from "./generated/themes/TableGroupRow.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-table-group-row",
+	slots: /** @lends sap.ui.webcomponents.main.TableGroupRow.prototype */ {
+		/**
+		 * Defines the text of the component.
+		 * <br>
+		 * <b>Note:</b> Although this slot accepts HTML Elements, it is strongly recommended that you only use text in order to preserve the intended design.
+		 *
+		 * @type {Node[]}
+		 * @slot
+		 * @public
+		 */
+		"default": {
+			type: Node,
+		},
+	},
+	properties: /** @lends sap.ui.webcomponents.main.TableGroupRow.prototype */ {
+		/**
```

Comment:
Does the GroupRow support selection modes? If not probably you can remove this prop.
Or it's done because of the code in Table.js

```js
 449: row.mode = this.mode;
```

Assigning the rows' "mode" field by the Table should not fail if the GroupRow does not have mode. And furthermore, it won't be re-rendered when the selection mode is change.

Reply:
The mode is needed as we need to know the exact number of columns, which is +1 in case of multi select mode.

Reply:
If you need it, then it's fine, thanks